.gitignore

```
@@ -2,3 +2,4 @@
 /coveralls-upload.json
 /phpunit.xml
 /vendor/
+.phpunit.result.cache
```

some repo seems have it, some repo not, if it needs to be added, it should has `/` prefix for consistency so it ensure pointed to root directory

```suggestion
/.phpunit.result.cache
```